<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cccccc;</string>
  </property>
  <widget class="QTextEdit" name="port_num">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>300</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ip_num">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>360</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>305</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_label">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>365</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #0000dd;</string>
   </property>
   <property name="text">
    <string>Server Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: red;</string>
   </property>
   <property name="text">
    <string>Server Terminate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bind_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00bbbb</string>
   </property>
   <property name="text">
    <string>Bind address</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_status">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gray;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>server off</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bind_status">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>370</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gray;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>not bind</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>port_label</zorder>
  <zorder>ip_label</zorder>
  <zorder>start_btn</zorder>
  <zorder>stop_btn</zorder>
  <zorder>port_num</zorder>
  <zorder>ip_num</zorder>
  <zorder>bind_btn</zorder>
  <zorder>server_status</zorder>
  <zorder>bind_status</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
